exec/java-exec/src/main/java/org/apache/drill/exec/vector/complex/fn/JsonReader.java

```diff
@@ -510,10 +517,13 @@ private void writeDataAllText(MapWriter map, FieldSelection selection,
       case VALUE_NUMBER_FLOAT:
       case VALUE_NUMBER_INT:
       case VALUE_STRING:
+        removeNotNullColumn(fieldName);
```

Comment:
But in the case, when null fields will be at the end of the file, it wouldn't work. So I removed it.